contracts/core/RebalancingSetToken.sol

```diff
@@ -171,21 +171,22 @@ contract RebalancingSetToken is
         require(msg.sender == manager);
 
         // New proposal cannot be made during a rebalance period
-        require(rebalanceState == State.Default);
+        require(rebalanceState != State.Rebalance);
 
         // Make sure enough time has passed from last rebalance to start a new proposal
         require(block.timestamp >= lastRebalanceTimestamp.add(rebalanceInterval));
 
+        // Check that new proposed Set is valid Set created by Core
+        address core = ISetFactory(factory).core();
```

Comment:
`core` doesn't look re-used. Insert `ISetFactory(factory).core()` directly into the next expression

Reply:
Yeah felt like that would be super messy though. Preferred the clarity.

Reply:
going for gas optimization at the contract level, extra memory consumed here